include/openssl/cms.h

```diff
@@ -313,16 +313,17 @@ int CMS_RecipientInfo_kari_get0_orig_id(CMS_RecipientInfo *ri,
                                         X509_ALGOR **pubalg,
                                         ASN1_BIT_STRING **pubkey,
                                         ASN1_OCTET_STRING **keyid,
-                                        X509_NAME **issuer,
-                                        ASN1_INTEGER **sno);
+                                        const X509_NAME **issuer,
+                                        const ASN1_INTEGER **sno);
```

Comment:
This is an incompatible change. Don't change it for a **

Reply:
Thanks @kroeckx for having a closer look at this commit and providing feedback.

Why should adding `const ` to a function parameter type be an incompatible change?

Reply:
Meanwhile I understand that adding `const` to a `**` parameter (as opposed to adding it to a `*` parameter) of a function can cause trouble (because the function could use it to return a pointer to `const` and then the caller could use it as pointer to mutable) and is therefore considered incompatible.

Reply:
It's not even the *change* that is incompatible; modern compilers will complain about having ```const foo **x``` in a function prototype (as opposed to ```foo const * const *x```, recalling that const binds to the left)

Reply:
Thanks @kaduk for this hint!
Using double `const` turned out to be the right solution (AFAICS) for situations where the function (for whatever reasons) takes a double pointer argument but does not modify the pointer nor the contents it points to.